src/ert/_c_wrappers/enkf/config/field_config.py

```diff
@@ -90,4 +68,6 @@ def save(self, run_path: Path, real_nr: int, ensemble: EnsembleReader):
 
 
 def field_transform(data: npt.ArrayLike, transform_name: str) -> npt.ArrayLike:
+    if transform_name is None:
```

Comment:
This is called once as:
```
data_transformed = field_transform(data, trans) if trans else data
```
and once as:
```
data_transformed = field_transform(data, trans)
```
suggest converging on one. I personally prefer checking before calling the function as I find that more readable, but that is personal preference.

Reply:
Also, the function `field_transform` states that `transform_name` is a `str` and not an `Optional`.
`field_config.py` is however not checked by `mypy`...

Reply:
will change transform_name to optional and check inside field_transform. I kind of agree with @oyvindeide about checking before calling, but since there is no enforcement of types in Python you will still need to check inside that function to prevent exceptions.